examples/tutorials/Introduction_to_Equivariant_Modelling_in_Materials_Science.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Please get rid of .ds_store modification and add this tutorial to the csvs for rendering</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4066/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>